workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt

```diff
@@ -119,7 +119,7 @@ class WorkflowViewStub @JvmOverloads constructor(
 
   override fun setId(@IdRes id: Int) {
     require(id == NO_ID || id != inflatedId) {
-      "id and inflatedId must be distinct"
+      "id must be distinct from inflatedId: ${resources.getResourceName(id)}"
```

Comment:
${resources.getResourceName(**inflatedId**)

Reply:
`id` is fine, it binds to the method parameter.

Reply:
ahh of course! they are the same :).